src/binarySearch.js

```diff
@@ -0,0 +1,22 @@
+const BinarySearch = (inputArray, searchValue) => {
+  let max = inputArray.length - 1
+  let min = 0
+  while (min <= max) {
+    let middle = (min + max) / 2
+    if(inputArray[middle] === searchValue) {
+      return middle
+    }
+    else {
+      if( inputArray[middle] < searchValue) {
+        min = inputArray[middle]
+      }
+      else {
+        max = inputArray[middle]
+      }
+    }
+  }
+
+  return -1
+}
```

Comment:
I don't have much to add to what @zubairnahmed said - your code is really clean and easy to read, and everything seems to work as intended. 👍 

It might be good to think about edge cases and write a few more tests - but none of the ones I tried off the top of my head broke anything.